src/dbload.jl

```diff
@@ -88,16 +91,23 @@ function __load_brusilovsky_comp__(name::AbstractString,
             )
 end
 
+"""
+    load(BrusilovskyEoSMixture; names[, comp_physics_db::ComponentDatabase, comp_eos_db::ComponentDatabase, mix_eos_db::MixtureDatabase])
+
+Returns `BrusilovskyEoSMixture` of `names` by loading parameters from
+
+- database of physical properties `physics_db`. Default is `CubicEoSDatabase.Data.martinez()`;
+- database of eos properties `eos_db`. Default is `CubicEoSDatabase.Data.brusilovsky_comp()`;
+- database of eos binary interaction parameters `mix_eos_db`. Default is `CubicEoSDatabase.Data.brusilovsky_mix()`.
+"""
 function load(::Type{<:BrusilovskyEoSMixture};
               names,
-              comp_physics_db::AbstractString,
-              comp_eos_db::AbstractString,
-              mix_eos_db::AbstractString)
-    # load physical parameters: crit temp, crit press, acentric
+              comp_physics_db::ComponentDatabase=Data.martinez(),
+              comp_eos_db::ComponentDatabase=Data.brusilovsky_comp(),
+              mix_eos_db::MixtureDatabase=Data.brusilovsky_mix())
```

Comment:
Default databases are which we used all the time. But, current implementation of `CubicEoSDatabase.Data` do not cache database collection, so each call `Data.martinez()` initialises new object. Same for the others.

I already added an issue on it.